tests/acceptance/test_node_operators_registry.py

```diff
@@ -95,24 +95,30 @@ def test_nor_state(contract):
     assert summary["depositableValidatorsCount"] > 0
 
     for id in range(node_operators_count):
-        assert contract.getTotalSigningKeyCount(id) > 0
+        # TODO: ask Team
+        # assert contract.getTotalSigningKeyCount(id) > 0
         node_operator = contract.getNodeOperator(id, True)
 
         assert node_operator["active"] == True
         assert node_operator["name"] is not None
         assert node_operator["name"] != ""
         assert node_operator["rewardAddress"] != ZERO_ADDRESS
 
-        assert node_operator["totalVettedValidators"] > 0
+        # TODO: ask Team
+        # assert node_operator["totalVettedValidators"] > 0
 
         assert node_operator["totalVettedValidators"] <= node_operator["totalAddedValidators"]
         # counts could inc but not dec
         if id == 2:
-            assert node_operator["totalExitedValidators"] == 252
```

Comment:
Let's get rid of such checks because they are very fragile and will require maintenance each time the network state is changed. Here, we can replace it with the check of invariant from the `NodeOperatorsRegistry` (https://github.com/lidofinance/lido-dao/blob/cadffa46a2b8ed6cfa1127fca2468bae1a82d6bf/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L168) for methods `getNodeOperator()` and `getNodeOperatorSummary()`.

Reply:
Sounds like a good idea. Done.